src/lib/pages/podcast-episodes/ui/specs/podcast-episodes-page.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/svelte';
+import { MockComponent } from '$lib/shared/lib/jest-hacks';
+import type { Podcast } from '$lib/shared/api';
+import { PodcastEpisodesPage } from '../..';
+
+jest.mock('~icons/ic/round-arrow-back', () => ({ default: MockComponent }), {
```

Comment:
Why did you use `MockComponent` in only one of the icons and left the rest as `null`?

Reply:
Because there was no reason to, but in the new update I made all of them `MockComponent`